### DISCUSSION TITLE
(Basketball) Is there a way to pull a full game log using the player object?

### DISCUSSION THREAD

**Discussion by SharKCS11** [cwendt94/espn-api#288] [Q&A]
First off, great tool... I could see many possible uses for it!
My question is, once we access a player, is there a way to get their list of stats for the whole season? For example, I instantiated my league object, and from a team's roster list, I have Player('Anthony Edwards').
Is there a way to get all of his stats for the 2021-22 season so far? Basically how it's listed here: https://www.espn.com/nba/player/gamelog/_/id/4594268/anthony-edwards

**Comment by cwendt94** [comment]
Thats a great question, currently you cannot get the break down of a players stats by game. You can see the players total stats for the current season and projected stats.

Football has a function called `player_info` that gets all of the players stats by game. I believe I can use the same API call for Basketball as well. I will have to look into the data because basketball stats are broken up by `scoringPeriodId` so its a little hard figuring out what week/team the player played.

**Reply by SharKCS11** [reply]
Got it, thanks for the response. Do you know what "scoringPeriodID" means exactly? Is that the dates that fall within a weekly matchup?

**Reply by cwendt94** [reply]
Yeah scoring period basically represents one day the NBA plays. So I think I can just map the stats by scoring period.

**Reply by cgkolias** [reply]
I wanted to add to this question something if I may! I was wondering, in the current API there is no way to pull stats based off each stats for basketball (FGM, FGA, Steals or whatever else)? Currently I only see points
